app/models/user.rb

```diff
@@ -57,6 +57,7 @@ module Permissions
   has_many :assignments
   has_many :uploads, class_name: CommonsUpload
   has_many :training_modules_users, class_name: 'TrainingModulesUsers'
+  has_one :user_profile, class_name: UserProfile, dependent: :destroy
```

Comment:
I think the `class_name` option is superfluous. Wouldn't rails automatically infer the model name from the association?

From rails docs,

> If the name of the other model cannot be derived from the association name, you can use the :class_name option to supply the model name.